lib/executors/Executor.js

```diff
@@ -297,10 +297,6 @@ define([
 				})
 				.finally(lang.bind(self, '_afterRun'))
 				.then(function () {
-					if (self._hasSuiteErrors) {
-						throw new Error('One or more suite errors occurred during testing');
```

Comment:
True, re-adding this so it exits with a non-zero exit code. I also had to fix serialization of fatal errors.